portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +38,96 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    /* LOADS HARD CODED ARRAYLIST
+    private List<String> strings;
+
+    //make array of hardcoded strings
+    @Override
+    public void init() {
+        strings = new ArrayList<>();
+        strings.add("Hello World!");
+        strings.add("My Name is Mannie");
+        strings.add("Testing 1, 2, 3");
+    }
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        //convert to json
+        String json = new Gson().toJson(strings);
+        
+        //set to return json
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+    */
+    //Loads Entities from Datastore
+
+    //build java struct/class for comments
```

Comment:
I like that you made a class for this. Good job! And good use of comments only where necessary